doc/modules/mixture.rst

```diff
@@ -211,49 +211,75 @@ Cons
-=================================
+Dirichlet Process: Infinite Gaussian mixture
+============================================
+
+The :class:`DirichletGaussianMixture` class implements a variant of the
+Gaussian mixture model with a variable (but bounded) number of components using
+the Dirichlet Process.
 
-The :class:`DPGMM` object implements a variant of the Gaussian mixture
-model with a variable (but bounded) number of components using the
-Dirichlet Process.
 This class doesn't require the user to choose the number of
 components, and at the expense of extra computational time the user
 only needs to specify a loose upper bound on this number and a
 concentration parameter.
 
-.. |plot_gmm| image:: ../auto_examples/mixture/images/sphx_glr_plot_gmm_001.png
+
+The examples bellow compare Gaussian mixture models with fixed number of
+components, to the Dirichlet Gaussian mixture models.
+
+.. figure:: ../auto_examples/mixture/images/sphx_glr_plot_gmm_001.png
    :target: ../auto_examples/mixture/plot_gmm.html
-   :scale: 48%
+   :align: center
+   :scale: 70%
+
+Here, a classical Gaussian mixture is fitted with 5 components on a dataset
+composed of 2 clusters. We can see that the Dirichlet Gaussian mixture is able
+to limit itself to only 2 components whereas the Gaussian mixture fits the data
+fit with too many components. Note that with very little observations, the
+Dirichlet Gaussian Mixture can take a conservative stand, and fit only one
+component.
 
-.. |plot_gmm_sin| image:: ../auto_examples/mixture/images/sphx_glr_plot_gmm_sin_001.png
+.. figure:: ../auto_examples/mixture/images/sphx_glr_plot_gmm_sin_001.png
    :target: ../auto_examples/mixture/plot_gmm_sin.html
-   :scale: 48%
+   :align: center
+   :scale: 65%
 
-.. centered:: |plot_gmm| |plot_gmm_sin|
+On that example, we are fitting a dataset not well-depicted by a Gaussian
+mixture. Adjusting the ``weight_concentration_prior``, parameter of the DPGMM
+controls the number of components used to fit this data.
 
+.. figure:: ../auto_examples/mixture/images/sphx_glr_plot_concentration_prior_002.png
+   :target: ../auto_examples/mixture/plot_concentration_prior.html
+   :align: center
+   :scale: 50%
 
-The examples above compare Gaussian mixture models with fixed number of
-components, to DPGMM models. **On the left** the GMM is fitted with 5
-components on a dataset composed of 2 clusters. We can see that the DPGMM is
-able to limit itself to only 2 components whereas the GMM fits the data fit too
-many components. Note that with very little observations, the DPGMM can take a
-conservative stand, and fit only one component. **On the right** we are fitting
-a dataset not well-depicted by a Gaussian mixture. Adjusting the `alpha`
-parameter of the DPGMM controls the number of components used to fit this
-data.
+The previous figure presents the resulting clusters computed by the Dirichlet
+Gaussian mixture for different values of ``weight_concentration_prior``. The
```

Comment:
Gaussian mixture with a Dirichlet process prior.

The "process" is important. 

The weights concentration prior of the BayesianGaussianMixture class is already based on Dirichlet distribution while the weights concentration prior of the DirichletGaussianMixture is based on a Dirichlet process.